terraform-provider/examples/cosmo/main.tf

```diff
@@ -0,0 +1,63 @@
+locals {
+  stages = {
+    dev  = {},
+    stg  = {},
+    prod = {}
+  }
+  subgraphs = {
+    "product-api" = {
+      routing_url = "http://product-api:3000/graphql"
+    },
+    "employees-api" = {
+      routing_url = "http://employees-api:3000/graphql"
+    }
+    "family-api" = {
+      routing_url = "http://family-api:3000/graphql"
+    },
+    "hobbies-api" = {
+      routing_url = "http://hobbies-api:3000/graphql"
+    },
+    "availability-api" = {
+      routing_url = "http://availability-api:3000/graphql"
+    },
+  }
+  stage_subgrahs = merge(flatten([
```

Comment:
This looks complicated as a setup. Can we simplify it by doing this in the provider?

Reply:
Sounds good!